model-client/integration-tests/docker-compose.yaml

```diff
@@ -0,0 +1,5 @@
+services:
+  mockserver:
+    image: mockserver/mockserver:5.14.0
```

Comment:
Too bad, we can't update the automatically with dependabot...

Reply:
I added the Docker Compose files to our Renovate-Ticket.
See https://issues.modelix.org/issue/MODELIX-613/Create-automatic-pull-requests-for-all-versions

Reply:
Also updated it to a never version.